### PR TITLE
[PWPA-1901] Implement score output file

### PR DESCRIPTION
**Pull request by gabrielantao**
In this PR I implement the outputs for the data generated with APB plugin, so it get the data from the ALFAsim result file and convets to the data for json. Only a few values are converted into the json data:

- temperature and pressure, final and initial for each annuli;
- temperature and pressure for the tubing;

The temperature for each wall wiil be filled in future (PWPA-2169)

PWPA-1901

### CODE REVIEW

File: src/alfasim_score/converter/alfacase/_tests/test_output_results/test_generate_output_results.json
Comment (ckiyono) [inline src/alfasim_score/converter/alfacase/_tests/test_output_results/test_generate_output_results.json:1]: Is this file an old test regression? There is no test with this name

**Review by ckiyono — APPROVED** [review]
Well done!

File: src/alfasim_score/converter/alfacase/_tests/test_output_results/test_generate_output_file_results.json [hunk 1]
```diff
@@ -0,0 +1,1414 @@
...
+          -273.15,
+          -273.15,
+          -273.15,
+          -273.15,
+          -273.15,
+          -273.15,
+          -273.15,
+          -273.15
+        ],
+        "final": [
```
Comment (noliveiraesss) [inline src/alfasim_score/converter/alfacase/_tests/test_output_results/test_generate_output_file_results.json:159]: Start and final temperatures match the values in the [spreadsheet](https://docs.google.com/spreadsheets/d/1E48nat1i_NH0nciEBzVyFZvhH96gbh9Ffu1bOJbJEyA/edit?gid=174790961#gid=174790961).

File: src/alfasim_score/converter/alfacase/_tests/test_output_results/test_generate_output_file_results.json [hunk 2]
```diff
@@ -0,0 +1,1414 @@
...
+          77.17743849273813,
+          78.6108492841899,
+          80.00203667369317,
+          80.86551653407275,
+          81.38448532358734
+        ]
+      },
+      "pressure": {
+        "start": [
+          3983.5801696854587,
```
Comment (noliveiraesss) [inline src/alfasim_score/converter/alfacase/_tests/test_output_results/test_generate_output_file_results.json:239]: Everything seems ok here also.

File: src/alfasim_score/converter/alfacase/_tests/test_output_results/test_generate_output_results.json [hunk 3]
```diff
@@ -0,0 +1,1414 @@
+{
+  "annuli": {
+    "0": {
+      "MD": [
```
Comment (noliveiraesss) [inline src/alfasim_score/converter/alfacase/_tests/test_output_results/test_generate_output_results.json]: The same as above.

File: src/alfasim_score/converter/alfacase/_tests/test_output_results/test_generate_output_file_results.json [hunk 4]
```diff
@@ -0,0 +1,1414 @@
...
+  "annuli": {
+    "0": {
+      "MD": [
+        2072.0,
+        2160.0,
+        2242.0,
+        2283.0,
+        2367.0,
+        2408.0,
+        2491.0,
```
[the remark this answers is not in the document]
Comment (noliveiraesss) [reply src/alfasim_score/converter/alfacase/_tests/test_output_results/test_generate_output_file_results.json:11]: TVD values should be:
{A2FDDA63-37DA-4647-8330-948DE1C5EE1C}

**Review by noliveiraesss — APPROVED** [review]
Great job!

**Review by carlosnewmar — APPROVED** [review]
Good job!

File: src/alfasim_score/converter/alfacase/_tests/test_output_results/test_generate_output_file_results.json (hunk 1 above)
Comment (gabrielantao) [reply src/alfasim_score/converter/alfacase/_tests/test_output_results/test_generate_output_file_results.json:159]: That's wonderful!

File: src/alfasim_score/converter/alfacase/_tests/test_output_results/test_generate_output_results.json
Comment (gabrielantao) [reply src/alfasim_score/converter/alfacase/_tests/test_output_results/test_generate_output_results.json:1]: Indeed! Thanks. Removed.

**Pull request merged**